packages/vsce/src/trees/CICSSessionTree.ts

```diff
@@ -28,15 +29,19 @@ export class CICSSessionTree extends TreeItem {
     super(profile.name, TreeItemCollapsibleState.Collapsed);
     this.children = [];
     this.contextValue = `cicssession.${profile.name}`;
+
     this.session = new imperative.Session({
-      type: "basic",
+      type: SessConstants.AUTH_TYPE_TOKEN,
+      storeCookie: true,
```

Comment:
I know I mentioned that perhaps we could move some of the LTPA token support to the CLI. 
However, it makes sense the way this is being done here since (as you mentioned on the last standup) this is all handled by Imperative.

Curious if the CICs plug-in could make use the the LTPA token when the CLI is in Daemon mode. 😋 
No need to worry about testing this, since people haven't asked for this feature yet 😋